internal/genny/ci/templates/dot-github/workflows/test.yml.tmpl

```diff
@@ -0,0 +1,45 @@
+name: test
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    {{ if eq .opts.DBType "postgres" -}}
+    services:
+      postgres:
+        image: postgres:9.6-alpine
+        env:
+          POSTGRES_DB: {{.opts.App.Name.File}}_test
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    {{- end }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ~1.18
+          cache: true
+      - name: setup
+        run: |
+          go get github.com/gobuffalo/buffalo/buffalo
```

Comment:
actually, it will not work. it should be replaced with `go install github.com/gobuffalo/cli/cmd/buffalo@latest` (and `go mod download` may not be required). I know you added this line copied from the others which is not maintained for a while :-(

Reply:
You are correct for both!
- I switched to this `go install` command
- I removed the `go mod download` 
🙇